lago/plugins/vm.py

```diff
@@ -27,6 +27,7 @@
 create an alternative implementation of the provisioning details for the VM,
 for example, using a remote libvirt instance or similar.
 """
+from copy import deepcopy
```

Comment:
See the discussion here: https://github.com/lago-project/lago/pull/346#pullrequestreview-5799308

Not 100% if its relevant, just checking.